### ISSUE TITLE
available simulations for CMIP5-CORDEX

### ISSUE THREAD

**Issue by JavierDiezSierra** [euro-cordex/joint-evaluation#26]
@larsbuntemeyer @jesusff

The script for downloading CMIP5-CORDEX-EUR-11 data for the evaluation experiment, including the variables ['tas', 'tasmin', 'tasmax', 'pr', 'psl', 'orog', 'sftlf'], is ready.

Before starting the download, I have checked the availability of both RCMs and variables.

Here is the list of available RCMs:

'CLMcom-ETH.COSMO-crCLIM-v1-1.v1'
'CLMcom.CCLM4-8-17.v1'
'CNRM.ALADIN53.v1'
'CNRM.ALADIN63.v1'
'DHMZ.RegCM4-2.v1'
'DMI.HIRHAM5.v1'
'GERICS.REMO2015.v1'
'ICTP.RegCM4-6.v1'
'IPSL.WRF381P.v1'
'KNMI.RACMO22E.v1'
'MOHC.HadREM3-GA7-05.v1'
'MPI-CSC.REMO2009.v1'
'RMIB-UGent.ALARO-0.v1'
'SMHI.RCA4.v1'

Minor issues encountered:

- IPSL.WRF381P.v1: Does not have monthly data, only daily.

- Variables not found:
  DHMZ.RegCM4-2.v1.psl
  IPSL.WRF381P.v1.orog
  IPSL.WRF381P.v1.sftlf
  RMIB-UGent.ALARO-0.v1.tasmin
  RMIB-UGent.ALARO-0.v1.tasmax
  RMIB-UGent.ALARO-0.v1.psl

**Comment by JavierDiezSierra** [comment]
I have just started to download cmip5-cordex data to: [/mnt/CORDEX_CMIP6_tmp/aux_data/cmip5-cordex](url)

**Comment by larsbuntemeyer** [comment]
@JavierDiezSierra The catalog.csv now holds only the CMIP5 data. Should we make two catalogs for CORDEX-CMIP5 and CORDEX-CMIP6? I see that you translated the attributes to CMIP6, that's good, maybe we should add a mip_era column to the catalog, so we can distinguish CMIP5 and CMIP6 data... Let me check how to handle sub collections...

**Comment by JavierDiezSierra** [comment]
@larsbuntemeyer Yes, I find both alternatives acceptable, a master catalog or two individual catalogs. Let me know if you need any help.

**Comment by larsbuntemeyer** [comment]
Agredd, i'll add a `mip_era` column!

**Comment by JavierDiezSierra** [comment]
@larsbuntemeyer Many thanks! Let me know once the new catalog is created (or I can create it if you prefer) with both CMIP5 and CMIP6 CORDEX simulations to include the CMIP5 analysis in the eobs.ipynb notebook. Currently, it is not working...

**Comment by JavierDiezSierra** [comment]
@larsbuntemeyer I have just updated the catalog to include both CMIP5 and CMIP6. I plan to commit the changes this morning. 

:)

**Comment by JavierDiezSierra** [comment]
Some CMIP5 simulations were removed for several issues:

- Corrupted files: '/mnt/CORDEX_CMIP6_tmp/aux_data/cordex-cmip5/CORDEX/output/EUR-11/ICTP/ECMWF-ERAINT/evaluation/r1i1p1/RegCM4-6/v1/mon/tas/v20190502'

- Error when combining by coords with xarray: '/mnt/CORDEX_CMIP6_tmp/aux_data/cordex-cmip5/CORDEX/output/EUR-11/DMI/ECMWF-ERAINT/evaluation/r1i1p1/HIRHAM5/v1/mon/tas'

**Comment by JavierDiezSierra** [comment]
@larsbuntemeyer I made several fixes to the HIRHAM5 CMIP5 fixed variables:
1- added missing projection
2- changed version (date) to match tas and pr

But I'm still not able to plot these simulations with the workflow.

Could you help me with this simulation?

**Comment by JavierDiezSierra** [comment]
@larsbuntemeyer  Is is already resolved: https://github.com/euro-cordex/joint-evaluation/commit/2e4d2bd725b7d71596f375514c2cb2d84de675ce

**Comment by larsbuntemeyer** [comment]
fyi: The [version date and variant label should be ignored](https://github.com/euro-cordex/evaltools/blob/5a0e632ce99c1509b895197b27d432e2bf4f14d3/evaltools/source.py#L187), when (fx) variables are aggregated (had the same problem before).
